tools/src/tokenize.rs

```diff
@@ -102,7 +102,59 @@ pub fn tokenize_c_like(string: &String, spec: &LanguageSpec) -> Vec<Token> {
 
     while cur_pos.get() < chars.len() {
         let (start, ch) = get_char();
-        if is_ident(ch) {
+
+        if ch == 'R' && peek_char() == '"' {
+            // Handle raw literals.
```

Comment:
Can you add a link to that C++ reference site in the comment? Probably easy to google, but it's still nice to have.